Ds3/Helpers/Strategies/ChunkStrategies/ReadRandomAccessChunkStrategy.cs

```diff
@@ -103,6 +106,20 @@ private TransferItem[] GetNextTransfers()
             .GetJobChunksReadyForClientProcessingSpectraS3(new GetJobChunksReadyForClientProcessingSpectraS3Request(this._jobId))
             .Match((ts, jobResponse) =>
             {
+                if (_lastAvailableChunks == null)
```

Comment:
i thought about doing it like that but i wanted it to be more clear, i dont mind to change it